src/runner/general_settings.cpp

```diff
@@ -68,6 +78,9 @@ void apply_general_settings(const json::JsonObject& general_configs) {
       }
     }
   }
+  if (json::has(general_configs, L"run_elevated", json::JsonValueType::Boolean)) {
+    run_as_elevated = general_configs.GetNamedBoolean(L"run_elevated");
```

Comment:
Could this be just `run_as_elevated = general_configs.GetNamedBoolean(L"run_elevated", false);` without using an `if`  block?